cost/google/unutilized_ip_addresses/google_unutilized_ip_addresses.pt

```diff
@@ -32,6 +32,14 @@ parameter "param_google_project" do
   allowed_pattern /^[0-9a-z:\.-]+$/
 end
 
+parameter "param_exclude_labels" do
+  category "User Inputs"
+  label "Exclusion Label Key:Value"
```

Comment:
same question on key:value

Reply:
It's key:value. https://cloud.google.com/resource-manager/docs/creating-managing-labels